src/plugins/controls/title/title.js

```diff
@@ -144,26 +103,32 @@ export default pluginFactory({
                     $timer,
                     hours: currentHours,
                     minutes: currentMinutes,
+                    seconds: currentSeconds,
                     stats,
                     unansweredQuestions: currentUnansweredQuestions,
                 } = this.titles[scope];
                 const time = moment.duration(remainingTime / precision, 'seconds');
                 const hours = time.get('hours');
                 const minutes = time.get('minutes');
+                const seconds = time.get('seconds');
                 const unansweredQuestions = stats && (stats.questions - stats.answered);
 
                 // check if notification should be updated
                 if (
                     currentHours !== hours
                     || currentMinutes !== minutes
+                    || currentSeconds !== seconds
                     || (unansweredQuestions && (unansweredQuestions !== currentUnansweredQuestions))
                 ) {
                     // update current timer state
-                    this.titles[scope].hours = hours;
-                    this.titles[scope].minutes = minutes;
-                    this.titles[scope].unansweredQuestions = unansweredQuestions;
-
-                    $timer.text(getTimerMessage(hours, minutes, unansweredQuestions));
+                    this.titles[scope] = {
+                        ...this.titles[scope],
```

Comment:
`...` is not allowed by eslint configuration . please use `Object.assign` instead